ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -416,9 +413,9 @@ objects:
         function update_image_registry() {
           while true; do
             sleep 10;
-            oc get configs.imageregistry.operator.openshift.io/cluster > /dev/null && break
+            oc --insecure-skip-tls-verify get configs.imageregistry.operator.openshift.io/cluster > /dev/null && break
           done
-          oc patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"managementState":"Managed","storage":{"emptyDir":{}}}}'
+          oc --insecure-skip-tls-verify patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"managementState":"Managed","storage":{"emptyDir":{}}}}'
```

Comment:
Why are you adding these?  Don't we have a kubeconfig with the CA for the Kube API?  If that CA doesn't match what the cluster serves us, I'd rather error out here instead of ignoring the mismatch.

Reply:
+1
I will remove it, it worked fine before.
I have added it because for some reason the "get nodes" fail for me locally without it.